### ISSUE TITLE
correlate should conjugate its second argument before multiplying its input (migrated from Trac #1120)

### ISSUE THREAD

**Issue by thouis** [thouis/numpy-trac-migration#2673]
Original ticket http://projects.scipy.org/numpy/ticket/1120
Reported 2009-06-01 by atmention:cournape, assigned to unknown.

correlate(x, y) computes sum(x \* y[k:]) for every k, instead of the usual definition of sum(x \* conj(y[:k])).

**Comment by thouis** [comment]
Comment in Trac by atmention:pv, 2009-07-12

Is this fixed by the new acorrelate function & documentation?

**Comment by thouis** [comment]
Comment in Trac by atmention:cournape, 2009-07-27

Yes, except I have renamed the underlying C function to PyArray_Correlate2 to follow numpy convention when fixing an exported C API which breaks backward compatibility, and the correlate function takes a keyword argument to handle new vs. old behavior, the old being deprecated for 1.4.0.

The documentation should be up to date, and I also added a note in the release notes.
